app/src/main/java/com/vci/vectorcamapp/incomplete_session/presentation/IncompleteSessionAction.kt

```diff
@@ -4,4 +4,5 @@ import java.util.UUID
 
 sealed interface IncompleteSessionAction {
     data class ResumeSession(val sessionId: UUID) : IncompleteSessionAction
+    data object ReturnToMain : IncompleteSessionAction
 }
```

Comment:
Sorry...I am too pedantic. Can you please rename this to ReturnToLandingScreen if you want this to return to the same screen?